README.rst

```diff
@@ -224,11 +224,23 @@ Render diagrams in untrusted jupyter notebooks
 
 The SVG format relies on a stylesheet, however in untrusted notebooks any
 stylesheets are stripped. As a workaround we support the PNG format. For this
-the `cairosvg`__ library is needed which requires the following additional
-setup steps on windows:
+the `cairosvg`__ library is needed which depends on additional binaries. They
+can be installed with the following commands based on your OS:
 
 __ https://pypi.org/project/CairoSVG/
 
+On linux
+""""""""
+
+Make sure to use the right package manager. For debian/ubuntu:
+
+.. code::
+
+    sudo apt-get install libcairo2
```

Comment:
The cairosvg installation instructions list two more packages, and also have instructions for macOS: https://cairosvg.org/documentation/#installation

What about linking there instead of trying to replicate the instructions here? It's a relatively big part of the README now, but it's only a small feature for us. This is a bit disproportionate.

Reply:
Excellent!